src/Auth/Login/Login.js

```diff
@@ -1,188 +0,0 @@
-import { useNavigate } from "react-router-dom";
```

Comment:
Always import into below manner. 
1. build in react packages 
2. third party react packages 
3. reusable react custom components 
4. helper functions.
5. redux, api services, utils 
6. css, scss, static images